tests/lib/fundamental-snaps.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
We might simplify this somewhat by calling this script as `names.sh`, and defining it as:
```
gadget_name=$(snap list | sed -n 's/^\(pc\|pi[23]\|dragonboard\) .*/\1/p')
kernel_name=$gadget_name-kernel

if [ "$kernel_name" = "pi3-kernel" ]; then
        kernel_name=pi2-kernel
fi
```
Then we don't need those assignments on every single place we use this file.